internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -24,9 +24,17 @@ func getValidMaturityStates() []MaturityStatus {
 }
 
 type KafkaInstanceType struct {
-	Id          string              `yaml:"id"`
-	DisplayName string              `yaml:"display_name"`
-	Sizes       []KafkaInstanceSize `yaml:"sizes"`
+	Id                     string                       `yaml:"id"`
+	DisplayName            string                       `yaml:"display_name"`
+	Sizes                  []KafkaInstanceSize          `yaml:"sizes"`
+	SupportedBillingModels []KafkaSupportedBillingModel `yaml:"supported_billing_models" validate:"min=1,unique=ID,dive"`
+}
+
+type KafkaSupportedBillingModel struct {
```

Comment:
I would follow the same pattern we used for the supported instance types (`KafkaInstanceType`) and call this `KafkaBillingModel` instead of `KafkaSupportedBillingModel`.
WDYT?

Reply:
:+1: . I will change the type but keep the name of the attribute as `SupportedBillingModels`, as that's the name for the yaml attribute. Let me know if that's ok

Reply:
Agree!

Reply:
updated